lib/rules/no-warning-comments.js

```diff
@@ -8,6 +8,8 @@
 const { escapeRegExp } = require("lodash");
 const astUtils = require("./utils/ast-utils");
 
+const CHAR_LIMIT = 30;
```

Comment:
I think this can be at least `40`, maybe even more. Thoughts?

Reply:
I think `40` is fine.